cypress/support/fragments/data_import/logs/logsViewAll.js

```diff
@@ -28,9 +30,9 @@ const verifyMessageOfDeteted = (quantity) => {
 };
 
 const columnName = {
-  status: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-status' })),
-  jobProfile: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-jobprofilename' })),
-  runBy: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-runby' }))
+  status: dataImportList.find(MultiColumnListHeader({ id:'list-column-status' })),
+  jobProfile: dataImportList.find(MultiColumnListHeader({ id:'list-column-jobprofilename' })),
+  runBy: dataImportList.find(MultiColumnListHeader({ id:'list-column-runby' }))
```

Comment:
MCLH also to const

Reply:
All MCLH are different. What MCLH do you mean?